### DISCUSSION TITLE
Можно ли не останавливать звук при переключении на другую вкладку?

### DISCUSSION THREAD

**Discussion by serggermob** [ilyhalight/voice-over-translation#502] [Q&A]
Останавливается звук при переключении на другую вкладку? Как продолжить озвучивание?
Не работает переключение видео в отдельное окно, поверх остальных. «Картинка в картинке» Возможно ли это?
Спасибо за приложение!

**Comment by NullVerdict** [comment]
> Останавливается звук при переключении на другую вкладку? Как продолжить озвучивание? Не работает переключение видео в отдельное окно, поверх остальных. «Картинка в картинке» Возможно ли это? Спасибо за приложение!

Напиши это в Issues, даже не указано в каком браузере проблема.

**Comment by FiLinX** [comment]
> Останавливается звук при переключении на другую вкладку
хотельсь бы уточнить  -  точно только звук? 
ну просто скорее всего это хромовский останов воспроизведения в фоновом плеере..
у хрома оооочень много сейчас стало функций на работу с памятью и выгрузку кеша вкладок..
чуть ли не с десяток флагов и ключей есть на управление всем этим..
и конечно же ещё и управление фоновыми плеерами в сторонних расширениях, во многих таких для ютуба есть функционал останова плееров на фоновых вкладках

**Comment by serggermob** [comment]
Спасибо за ответы, стоит хром 109.0.5414.120 , windows 7.
Попробовал вкл.выкл флаг - Calculate window occlusion on Windows безрезультатно
Подскажите пожалуйста конкретнее, что можно сделать.
